R/mcmc.R

```diff
@@ -62,14 +54,16 @@ fit_mcmc <- function(
     group,
     subjid,
     visit,
-    same_cov,
-    n_imputations,
-    burn_in,
-    burn_between,
-    verbose = TRUE,
-    seed = NULL
+    method
```

Comment:
I'm suprised this didn't require any changes to our unit tests. Do we not have anything for fit_mcmc ?

Reply:
I just checked that we have some tests for fit_mcmc on nightly mode.. I will make changes accordingly

Reply:
ok, just pushed the changes